### ISSUE TITLE
"Can't parse neither ipython kernelspecs nor jupyter kernelspecs" error

### ISSUE THREAD

**Issue by sbromberger** [nteract/hydrogen#206]
I'm running into this issue now. I upgraded to Atom 1.6.0, and am using Hydrogen 0.7,2 with zmq 4.1.4 and jupyter 4.1.0. Despite the error in Atom, I can do:

```
seth@schroeder:~$ jupyter kernelspec list --json
{
  "kernelspecs": {
    "julia-0.4": {
      "resource_dir": "/Users/seth/Library/Jupyter/kernels/julia-0.4",
      "spec": {
        "language": "julia",
        "display_name": "Julia 0.4.1-pre",
        "env": {},
        "argv": [
          "/usr/local/julia-latest/bin/julia",
          "-i",
          "-F",
          "/Users/seth/.julia/v0.4/IJulia/src/kernel.jl",
          "{connection_file}"
        ]
      }
    },
    "python2": {
      "resource_dir": "/Users/seth/Library/Python/2.7/lib/python/site-packages/ipykernel/resources",
      "spec": {
        "language": "python",
        "display_name": "Python 2",
        "env": {},
        "argv": [
          "/usr/bin/python",
          "-m",
          "ipykernel",
          "-f",
          "{connection_file}"
        ]
      }
    }
  }
}
```
and the same for ipython (but the latter gives me a deprecation warning).

How do I fix this?

**Comment by sbromberger** [comment]
Hm. it works when I launch atom from the command line. But I thought this was fixed via #176?

**Comment by n-riesco** [comment]
On 18/03/16 04:12, Seth Bromberger wrote:
> Hm. it works when I launch atom from the command line. But I thought this was fixed via #176 <https://github.com/nteract/hydrogen/issues/176>?

This is Atom's issue https://github.com/atom/atom/pull/11054 .

I can see that although the fix is not in Atom 1.6.0 it's now present in 1.7.0-beta0.

BTW, is this issue limited to OS X?

**Comment by sbromberger** [comment]
I'm using OSX (El Cap 10.11.3) and can't speak to any other OS. Sorry.

**Comment by n-riesco** [comment]
@sbromberger If you update your Atom installation, please, could you post here whether this bug is still present?

**Comment by n-riesco** [comment]
I'm closing this issue because it should be fixed since Atom v1.8.0
